app/models/scheduled_message.rb

```diff
@@ -2,4 +2,18 @@ class ScheduledMessage < ActiveRecord::Base
   validates :body, presence: true
   validates :days_after_start, presence: true
   validates :title, presence: true
+
+  before_save :escape_specific_characters
```

Comment:
What about encoding on send (in the service class) rather than in a callback? I try to eliminate ALL callbacks in Rails when possible. Good blog post for ref: http://samuelmullen.com/2013/05/the-problem-with-rails-callbacks/